library/src/main/java/com/google/android/exoplayer/audio/AudioTrack.java

```diff
@@ -1280,8 +1280,8 @@ public AudioTrackUtil(boolean isLatencyQuirkEnabled,
       this.getLatencyMethod = getLatencyMethod;
       try {
         getTimestamp = AudioTrack.class.getMethod("getTimestamp", AudioTimestamp.class);
-      } catch (Exception e) {
-        // There's no guarantee this method exists. Do nothing.
+      } catch (Throwable e) {
```

Comment:
Not quite. The current handling prior to this commit correctly handles the missing `getTimeStamp` method as it throws the aforementioned exception. However if the `AudioTimestamp` class itself is not present, then we miss handling a thrown `NoClassDefFoundError` from the runtime as this doesn't derive from `Exception`.